### PR TITLE
Fix issue #222

### PR DESCRIPTION
**Pull request by SeRi0uS007**
Trying to fix raw string issue for GetPriceValueAsInt
#222

### CODE REVIEW

**Comment by SeRi0uS007** [comment]
I will check with amounts >1000
I think I forgot about it

**Comment by xPaw** [comment]
You can use `v_currencyformat(1234567890, 'UAH')` to loop some random values over all currencies to test if you want.

**Comment by SeRi0uS007** [comment]
Yep, I definitely missed prices over 1000.
In addition, the previous version did not consider that a whole part of the price could be without a separator at all.

Now, it seems, I have considered everything:
img

File: code.user.js [hunk 1]
```diff
@@ -2566,6 +2566,17 @@
             }
         }
 
+        function getPriceValueAsInt(listing) {
+            // Match number part from any currency format
+            const results = listing.match(/\d[0-9 .,]*(\.|,\d{0,2})?/gm);
```
Comment (xPaw) [inline code.user.js:2571]: `/\d[0-9 .,]+/` I think this works just as well?

Comment (SeRi0uS007) [reply code.user.js]: Yep and looks more elegant

**Comment by Nuklon** [comment]
Is this done now/working?

**Comment by SeRi0uS007** [comment]
@Nuklon , yep
It's working

**Pull request merged**